### PR TITLE
A few minor changes

### PR DESCRIPTION
**Pull request by OliverBrotchie**
- Added space in between links.
- Moved to SCSS for nesting and minification.
- Added prettier config file to standardize formatting.
- Changed to standardized resource file structure 'dir/lib/subfolders/ect'.

### CODE REVIEW

**Comment by humaidq** [comment]
Hey, thank you for the PR!

I'll summarise the changes made, and include my thoughts on them.

- Change indentation from two to four spaces
  - HTML is a heavily nested markup language, so we try to keep the indentation minimal.
- Use SCSS instead of CSS
  - GitHub pages doesn't support SCSS without using Jekyll, so we would have to either use Jekyll or switch our hosting to Netlify. And SCSS seems to be unnecessary for a simple HTML page with basic styling, so we wouldn't make much use of the nesting features.
- Adding space between links
  - Probably a personal preference, but I find it easier to read when the `<li>` and `<a>` are on one line.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "endOfLine": "crlf",
```
Comment (alakbarz) [inline .prettierrc:2]: Why is the EOL set to CRLF? Also why is tabWidth set to 4? Any specific reasons for doing so? (btw first time using this review feature of Github so bear with me please)

**Comment by OliverBrotchie** [comment]
> * Change indentation from two to four spaces
>   
>   * HTML is a heavily nested markup language, so we try to keep the indentation minimal.
> * Use SCSS instead of CSS
>   
>   * GitHub pages doesn't support SCSS without using Jekyll, so we would have to either use Jekyll or switch our hosting to Netlify. And SCSS seems to be unnecessary for a simple HTML page with basic styling, so we wouldn't make much use of the nesting features.
> * Adding space between links
>   
>   * Probably a personal preference, but I find it easier to read when the `<li>` and `<a>` are on one line.

Indentation now set to 2 spaces. ✔️

SCSS is a compiled language and hence you only need to link the css file. You may be correct that its not necessary but there are quite a few reasons why something like this is useful:
- It compiles to minified CSS.
- Mixins reduce a huge amount of code repetition.
- It allows for modularisation of your css, making it more maintainable and efficient.

I meant I added margin in between the links on the page, the <li><a> space was added automatically with the prettier format (although that is a setting you can change).

File: .prettierrc (hunk 1 above)
Comment (OliverBrotchie) [reply .prettierrc:2]: - CRLF is just the carriage return character.
- I just copied my standard code prettier config, its been changed to 2. 
Btw you need the prettier extension installed in VScode (also make sure to have the format on save option installed).

**Comment by humaidq** [comment]
Partially merged by 4088f91

**Pull request closed without merging**